governance/default.go

```diff
@@ -996,6 +998,7 @@ type governanceJSON struct {
 func (gov *Governance) toJSON(num uint64) ([]byte, error) {
 	ret := &governanceJSON{
 		BlockNumber:     num,
+		ChainConfig:     gov.ChainConfig,
```

Comment:
Do we need this line? It seems that `Governance.ChainConfig` is set by `NewGovernance`, and it is not unmarshalled by `UnmarshalJSON`.

Reply:
Without this line, WriteGovernanceState would write `{... "chainConfig":null ... }` to the miscDB.
If v1.9 node read such DB, v1.9 `UnmarshalJSON` do unmarshal ChainConfig into memory. This caused the panic.

Reply:
Oh, I see. v1.9 override chainConfig again with governance JSON data in `ReadGovernanceState`. Thank you for the explanation.